### DISCUSSION TITLE
How to delete export files after export?

### DISCUSSION THREAD

**Discussion by JF-626164** [Power-Components/livewire-powergrid#206] [Q&A] [answered]
Is there a way to delete/prevent files that were exported from remaining in source control? I checked the documentation and didn't see a way.

**Comment by luanfreitasdev** [comment, marked as answer]
At the moment it doesn't exist. But it is interesting to put an option in the backend allowing or not the deletion right after the export.

**Reply by JF-626164** [reply]
Okay. I would recommend adding it in, that way on the server side, there isn't a thousand export files that need to be managed.

**Reply by luanfreitasdev** [reply]
If you want, you can submit a PR too.

**Reply by trippo** [reply]
I created a pull request https://github.com/Power-Components/livewire-powergrid/pull/223

**Reply by JF-626164** [reply]
Awesome!
